### DISCUSSION TITLE
Cargo system

### DISCUSSION THREAD

**Discussion by SOF-Ice** [Vdauphin/HeartsAndMinds#1441] [Question and answer] [answered]
Hi Vdauphin,

I was reading the topic about ace cargo and H&M cargo from Razor,
And when i search core/fnc/eh/server.sqf i don't see same lines ...
i guess you update it.

My question is: can i modify an ace cargo vehicle in EDEN and and keep the modification active even if it is destroyed and repaired during the game ?

**Comment by Vdauphin** [comment, marked as answer]
Hello,

> can i modify an ace cargo vehicle in EDEN and and keep the modification active even if it is destroyed and repaired during the game ?

No because you want to apply this behavior to all object with the same class or parent to it
Use `CBA_fnc_addClassEventHandler` as explained here https://github.com/Vdauphin/HeartsAndMinds/discussions/1242#discussioncomment-1654801 

Cheers
